### PR TITLE
EXIOBASE 2019 for Norway and Brazil

### PR DESCRIPTION
**Pull request by AnastasiaLobanova**
**Is this a new addition or an update to an existing factor?**

- [x] New
- [ ] Update

**What category or categories does it affect?**

Multiple

**What?**

EXIOBASE 2019 for Norway and Brazil

**Why?**

Part of the spend-based data ingress

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -1862,9 +2007,13 @@ Energy,Fuel,fuel_type_other_petroleum_gas_net-fuel_use_na,Other Petroleum Gas -
...
+Energy,Fuel,fuel_type_paraffin_waxes-fuel_use_na,Paraffin waxes,EUR,0.0913,0.0874,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,unknown,EXIOBASE,2021,2019-2021,2019,BR,o|h,Climatiq,2022/03/17,Emission intensity of supply chain in EUR spend on: paraffin waxes. Retrieved from the EXIOBASE v3 model outputs for products. These factors were calculated based on 2019 data. CO2 equivalent factors incorporate emissions from land use; constituent gases have not been included as they do not. The LCA boundaries of these factors are not defined by the source. These factors include effects of international trade. WARNING: This factor differs substantially from similar data points and could be erroneous. This factor exhibits suspicious pattern and could be erroneous.It is included to retain consistency with the source.,https://zenodo.org/record/5589597#.Yh9_Zi8w1ao
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:2015]: why is this value marked as "o" but not value lower than it?

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -2680,79 +2879,127 @@ Materials,Glass and Glass Products,glass_products-type_glass_and_glass_products,
...
+Materials,Metals,metals-type_precious_metals,Precious metals,EUR,0.0992,0.0975,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,unknown,EXIOBASE,2021,2019-2021,2019,NO,o,Climatiq,2022/03/17,Emission intensity of supply chain in EUR spend on: precious metals. Retrieved from the EXIOBASE v3 model outputs for products. These factors were calculated based on 2019 data. CO2 equivalent factors incorporate emissions from land use; constituent gases have not been included as they do not. The LCA boundaries of these factors are not defined by the source. These factors include effects of international trade. WARNING: This factor differs substantially from similar data points and could be erroneous. It is included to retain consistency with the source.,https://zenodo.org/record/5589597#.Yh9_Zi8w1ao
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:2909]: why is this value marked as "o" but not values lower than this?

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -2680,79 +2879,127 @@ Materials,Glass and Glass Products,glass_products-type_glass_and_glass_products,
...
+Materials,Mined Materials,mined_materials-type_natural_gas_liquids,Natural gas liquids,EUR,11.791,11.543,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,unknown,EXIOBASE,2021,2019-2021,2019,BR,,Climatiq,2022/03/17,Emission intensity of supply chain in EUR spend on: natural gas liquids. Retrieved from the EXIOBASE v3 model outputs for products. These factors were calculated based on 2019 data. CO2 equivalent factors incorporate emissions from land use; constituent gases have not been included as they do not. The LCA boundaries of these factors are not defined by the source. These factors include effects of international trade.,https://zenodo.org/record/5589597#.Yh9_Zi8w1ao
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:2963]: shouldn't this and 2958, 228, 69, 62 be marked as "o"?

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -2886,6 +3155,8 @@ Organizational Activities,Government Activities,organizational_activities-type_l
...
+Organizational Activities,Government Activities,organizational_activity-type_public_administration_defense_services_compulsory_social_security_services,Public administration and defense services/compulsory social security services,EUR,0.0913,0.0874,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,unknown,EXIOBASE,2021,2019-2021,2019,BR,h,Climatiq,2022/03/17,Emission intensity of supply chain in EUR spend on: public administration and defence services/compulsory social security services. Retrieved from the EXIOBASE v3 model outputs for products. These factors were calculated based on 2019 data. CO2 equivalent factors incorporate emissions from land use; constituent gases have not been included as they do not. The LCA boundaries of these factors are not defined by the source. These factors include effects of international trade. WARNING: This factor exhibits suspicious pattern and could be erroneous. It is included to retain consistency with the source.,https://zenodo.org/record/5589597#.Yh9_Zi8w1ao
```
Comment (maryarbab) [inline OpenEmissionFactorsDB.csv:3158]: shouldn't this marked as o|h?

**Review by maryarbab — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv [hunk 5]
```diff
@@ -1650,9 +1779,13 @@ Energy,Fuel,fuel_type_kerosene-fuel_use_stationary_combustion,Kerosene,MMBTU,not
...
+Energy,Fuel,fuel_type_kerosene_type_jet_fuel-fuel_use_aviation,Kerosene type jet fuel,EUR,0.676,0.6409,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,unknown,EXIOBASE,2021,2019-2021,2019,NO,h,Climatiq,2022/03/17,Emission intensity of supply chain in EUR spend on: kerosene type jet fuel. Retrieved from the EXIOBASE v3 model outputs for products. These factors were calculated based on 2019 data. CO2 equivalent factors incorporate emissions from land use; constituent gases have not been included as they do not. The LCA boundaries of these factors are not defined by the source. These factors include effects of international trade. WARNING: This factor exhibits suspicious pattern and could be erroneous. It is included to retain consistency with the source.,https://zenodo.org/record/5589597#.Yh9_Zi8w1ao
```
[the remark this answers is not in the document]
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:1788]: Yes, they are not "identical" but if you look at the other fuels for Norway the values (e.g. rows 1818, 1912, 1932, 1783) are very similar to each other, so I have marked it as suspicious

File: OpenEmissionFactorsDB.csv [hunk 6]
```diff
@@ -2085,6 +2242,8 @@ Energy,Heat and Steam,heat-and-steam-type_purchased,Steam and Heat,MMBTU,not-sup
...
+Energy,Heat and Steam,heat_and_steam-type_steam_hot_water_supply_services,Steam and hot water supply (services),EUR,0.1298,0.1241,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,unknown,EXIOBASE,2021,2019-2021,2019,BR,o|h,Climatiq,2022/03/17,Emission intensity of supply chain in EUR spend on: steam and hot water supply (services). Retrieved from the EXIOBASE v3 model outputs for products. These factors were calculated based on 2019 data. CO2 equivalent factors incorporate emissions from land use; constituent gases have not been included as they do not. The LCA boundaries of these factors are not defined by the source. These factors include effects of international trade. WARNING: This factor differs substantially from similar data points and could be erroneous. This factor exhibits suspicious pattern and could be erroneous.It is included to retain consistency with the source.,https://zenodo.org/record/5589597#.Yh9_Zi8w1ao
```
[the remark this answers is not in the document]
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:2245]: What do you mean in other instances? The values for waste management efs?

File: OpenEmissionFactorsDB.csv (hunk 1 above)
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:2015]: There is not lower value for "Parafin waxes" across other region - or which value do you mean?

File: OpenEmissionFactorsDB.csv (hunk 2 above)
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:2909]: this is the lowest value for "Precious Metals" across 49 regions

File: OpenEmissionFactorsDB.csv (hunk 3 above)
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:2963]: Across the 49 regions there are several values that are around ~10 kgCO2e for different regions, they are kind of cluster of values (so technically they are not outliers anymore) and that's why they were not marked as outliers by DBSCAN. 

same applies for lines 2958, 228, 69 and 62. We can discuss of course if we still think they might be outliers

File: OpenEmissionFactorsDB.csv (hunk 4 above)
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:3158]: This value doesn't seem as an outlier if you look across 49 regions.. I will check the "h" though

File: OpenEmissionFactorsDB.csv [hunk 7]
```diff
@@ -2761,30 +3008,48 @@ Materials,Organic Products,organic_products-type_compost_derived_from_garden_was
...
+Materials,Other Materials,chemicals-type_charcoal,Charcoal,EUR,0.9064,0.8926,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,unknown,EXIOBASE,2021,2019-2021,2019,BR,,Climatiq,2022/03/17,Emission intensity of supply chain in EUR spend on: charcoal. Retrieved from the EXIOBASE v3 model outputs for products. These factors were calculated based on 2019 data. CO2 equivalent factors incorporate emissions from land use; constituent gases have not been included as they do not. The LCA boundaries of these factors are not defined by the source. These factors include effects of international trade.,https://zenodo.org/record/5589597#.Yh9_Zi8w1ao
```
[the remark this answers is not in the document]
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:3027]: Yes why not - I will change the category to Chemical Products

[the remark this answers is not in the document]
Comment (patstirling) [reply OpenEmissionFactorsDB.csv:3027]: > Yes why not - I will change the category to Chemical Products

I would keep it where it is, just change the ID to "other_materials" - charcoal isn't really a "chemical product" per se (no more than other wood-based products)

File: OpenEmissionFactorsDB.csv [hunk 8]
```diff
@@ -15,51 +19,81 @@ Agriculture/Hunting/Forestry/Fishing,Arable Farming,arable_farming-type_greenhou
...
+Agriculture/Hunting/Forestry/Fishing,Arable Farming,arable_farming-paddy_rice,Paddy Rice,EUR,0.1998,0.1961,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,unknown,EXIOBASE,2021,2019-2021,2019,NO,,Climatiq,2022/03/17,Emission intensity of supply chain in EUR spend on: paddy rice. Retrieved from the EXIOBASE v3 model outputs for products. These factors were calculated based on 2019 data. CO2 equivalent factors incorporate emissions from land use; constituent gases have not been included as they do not. The LCA boundaries of these factors are not defined by the source. These factors include effects of international trade.,https://zenodo.org/record/5589597#.Yh9_Zi8w1ao
```
[the remark this answers is not in the document]
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:26]: I also found it suspicious but there are 2 other EFs that are comparable to this values. I think it is up to our expert judgement to decide here whether these are outliers or not.

File: OpenEmissionFactorsDB.csv [hunk 9]
```diff
@@ -15,51 +19,81 @@ Agriculture/Hunting/Forestry/Fishing,Arable Farming,arable_farming-type_greenhou
...
 Agriculture/Hunting/Forestry/Fishing,Livestock Farming,livestock_farming-type_meat_animals_not_elsewhere_specified,Meat animals (not elsewhere specified),EUR,143.174,144.8726,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,unknown,EXIOBASE,2021,2019-2021,2019,US,,Climatiq,2022/03/17,Emission intensity of supply chain in EUR spend on: meat animals (not specified in other EXIOBASE emission factors). Retrieved from the EXIOBASE v3 model outputs for products. These factors were calculated based on 2019 data. CO2 equivalent factors incorporate emissions from land use; constituent gases have not been included as they do not. The LCA boundaries of these factors are not defined by the source. These factors include effects of international trade. WARNING: This factor differs substantially from similar data points and could be erroneous. It is included to retain consistency with the source.,https://zenodo.org/record/5589597#.Yh9_Zi8w1ao
```
[the remark this answers is not in the document]
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:68]: yes

File: OpenEmissionFactorsDB.csv [hunk 10]
```diff
@@ -2680,79 +2879,127 @@ Materials,Glass and Glass Products,glass_products-type_glass_and_glass_products,
...
+Materials,Metals,metals-type_lead_zinc_tin_products,Lead/zinc and tin and products thereof,EUR,10.594,10.5813,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,unknown,EXIOBASE,2021,2019-2021,2019,BR,o,Climatiq,2022/03/17,Emission intensity of supply chain in EUR spend on: lead/zinc and tin and products thereof. Retrieved from the EXIOBASE v3 model outputs for products. These factors were calculated based on 2019 data. CO2 equivalent factors incorporate emissions from land use; constituent gases have not been included as they do not. The LCA boundaries of these factors are not defined by the source. These factors include effects of international trade. WARNING: This factor differs substantially from similar data points and could be erroneous. It is included to retain consistency with the source.,https://zenodo.org/record/5589597#.Yh9_Zi8w1ao
```
[the remark this answers is not in the document]
Comment (AnastasiaLobanova) [reply OpenEmissionFactorsDB.csv:2898]: From "statistical" point of view, when you look at other values it is clearly an outlier.

**Review by maryarbab — APPROVED** [review]

**Review by maryarbab — APPROVED** [review]

**Pull request merged**